nix/tests/default.nix

```diff
@@ -19,14 +19,15 @@
 let
   inherit (pkgs) lib;
   cleanSrc = gitignore-nix.gitignoreSource src;
+  noCross = x: if pkgs.stdenv.hostPlatform == pkgs.stdenv.buildPlatform then x else {};
 in
 pkgs.recurseIntoAttrs {
   shellcheck = pkgs.callPackage ./shellcheck.nix { src = cleanSrc; };
 
-  stylishHaskell = pkgs.callPackage ./stylish-haskell.nix {
+  stylishHaskell = noCross (pkgs.callPackage ./stylish-haskell.nix {
```

Comment:
Same question: let's just not ask for `tests` in ci.nix when we're cross.

Reply:
I still think this might be simpler? Or is the issue that it won't even evaluate? But that ought to also be okay if we just didn't pull in the test attrset if we're cross...